src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,22 @@
+import { jwtService } from '../service/jwt.service.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    res.sendStatus(401);
+
+    return;
+  }
+
+  const userData = jwtService.verify(token);
```

Comment:
Consider assigning `userData` to `req.user` or a similar property on the request object. This allows downstream middleware or route handlers to access the authenticated user's data.